src/main/java/com/enterprise/redcord/dto/User.java

```diff
@@ -12,5 +13,13 @@
     @SerializedName("email") private String email;
     @SerializedName("status") private String status;
 
+
+    public void setPassword(String userPassword) {
+        BasicPasswordEncryptor passwordEncryptor = new BasicPasswordEncryptor();
+        String encryptedPassword = passwordEncryptor.encryptPassword(userPassword);
+        this.password = encryptedPassword;
+
+    }
```

Comment:
Is this used anywhere?

This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality .
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.